README.md

```diff
@@ -13,13 +13,23 @@ months: ['Janeiro', 'Fevereiro', 'Março', 'Abril', 'Maio', 'Junho',
          'Julho', 'Agosto', 'Setembro', 'Outubro', 'Novembro', 'Dezembro'],
 textArrows: {previous: '<', next: '>'},
 eventTitle: 'Eventos',
+eventDisplayDateOnEventList : true,
 url: '',
 events: [
     {title: 'Event Title 1', description: 'Description 1', datetime: new Date(2016, 0, 12, 17)},
     {title: 'Event Title 2', description: 'Description 2', datetime: new Date(2016, 0, 23, 16)}
 ],
 firstDayOfWeek: 0
 ```
+####Event listeners
```

Comment:
I opened a PullRequest  #16 to fix to some issues in the this file.

Just add an space on the title to render it well. See #16 for examples